R/weightingdialog.R

```diff
@@ -0,0 +1,126 @@
+#' \code{WeightsDialog}
+#' @description Code for the Displayr Weighting dialog box.
+#' @param categorical.variables An optional list or data frame of categorical adjustment variables.
+#' @param categorical.targets The target probabilities for each category listed in \code{categorical.variables}
+#' @param numeric.variables An optional list or data frame of categorical adjustment variables.
+#' @param numeric.targets the target mean for each numeric variable in numeric.variables
+#' @param lower A lower bound weight value (not guaranteed to be achieved).
+#' @param upper An upper bound weight value (not guaranteed to be achieved).
+#' @param calfun The calibration function: Raking (Default), Linear, Quadratic.
+#' @param input.weight An optional weight variable; if supplied, the created weight is created to be as close
+#' to this input.weight as possible
+#' @param force.to.n Force the sum of weights to equal the sample size.
+#'
+#' @return numeric A vector of weights
+#' @export
+
+
+WeightingDialog <- function(categorical.variables = NULL,
```

Comment:
Will this code ever be viewed/edited in the R Code editor in Displayr? If so, is there a better name for this function without the word "Dialog"? (Similarly, for `computeWeightsDialog`).